node/components/network/src/testonly.rs

```diff
@@ -78,15 +78,19 @@ pub(crate) async fn forward(
     let _ = io::shutdown(ctx, &mut write).await;
 }
 
-/// Node instance, wrapping the network actor state and the
+/// Node instance, wrapping the network component state and the
 /// events channel.
 pub struct Instance {
     /// State of the instance.
-    pub(crate) net: Arc<Network>,
+    pub net: Arc<Network>,
     /// Termination signal that can be sent to the node.
-    pub(crate) terminate: channel::Sender<()>,
```

Comment:
Instance has a public method `terminate()` to interact with this field, why do you want to make this field public? In fact, is there anything blocking it from being private?

Reply:
I need the following two fields to be public and clippy complains about mixed visibility in structs. I'll add an exception here.